hack/importAzure.sh

```diff
@@ -44,6 +44,7 @@ AZURE_IMAGE_NAME="${AZURE_IMAGE_NAME:-upload-target}"
 AZURE_IMAGE_OFFER="${AZURE_IMAGE_OFFER:-constellation}"
 AZURE_IMAGE_DEFINITION="${AZURE_IMAGE_DEFINITION:-constellation}"
 AZURE_SKU="${AZURE_SKU:-constellation-coreos}"
+AZURE_SECURITY_TYPE="${AZURE_SECURITY_TYPE:-ConfidentialVmSupported}"
```

Comment:
```suggestion
AZURE_SECURITY_TYPE="${AZURE_SECURITY_TYPE:-TrustedLaunch}"
```

I would choose a working security type for TrustedLaunch as default. `ConfidentialVmSupported` seems to only work for CVMs (just like security type `ConfidentialVM`).

Reply:
I wasn't sure who else uses this script and thus what should be the default. Is usage of this script with CVMs outdated and is it safe to change the default behavior?

Reply:
The script is only used for trusted launch. I also searched around our repo and this is the only reference. Re-importing images in other scenarios is not required.